pkgs/development/tools/misc/findnewest/default.nix

```
@@ -0,0 +1,22 @@
+{ stdenv, fetchFromGitHub, autoreconfHook }:
+
+stdenv.mkDerivation rec {
+  name = "findnewest-${version}";
+  version = "0.3";
+
+  src = fetchFromGitHub {
+    owner = "0-wiz-0";
+    repo = "findnewest";
+    rev = name;
+    sha256 = "1x1cbn2b27h5r0ah5xc06fkalfdci2ngrgd4wibxjw0h88h0nvgq";
+  };
+
+  buildInputs = [ autoreconfHook ];
```

```suggestion
  nativeBuildInputs = [ autoreconfHook ];
```
Probably `autoreconfHook` is only needed at compile time.